app/views/requests/index.html.erb

```diff
@@ -3,14 +3,15 @@
     <h5>Messages</h5>
   </div>
   <div class="container my-3">
-    <%# if @requests is not empty, show them. Otherwise, "You don't have any requests." %>
-    <% if @requests %>
+    <%# if @requests is empty, "You don't have any requests." Show the requests otherwise. %>
+    <% if @requests.nil? || @requests.all? { |r| r.messages.last.nil? } %>
```

Comment:
Only problem with this is that if I start a chat, but don't send a message, I can't find that chat in my messages. But on the item it will still say 'go back go chat' as if a chat was started. I think even if there are no messages but there is a request, that it should pop up in chats. 
![image](https://user-images.githubusercontent.com/85902450/171511884-e6d542f4-f105-476b-926e-3ecf1bc171e6.png)

Reply:
Yeah I thought about that too...but I think there's no point for `giver` to see a request from `receiver` with empty message in the chats page. I wanted for `giver` to see the chat only when there's a message sent by `receiver`. Does that make sense to you? *0* For `receiver` to `go back to chat` in Items#show page is fine i guess...Let's discuss later!